gyroscope/Gyroscope_onerror-manual.https.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Gyroscope Test: onerror</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="https://www.w3.org/TR/gyroscope/">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/generic-sensor/generic-sensor-tests.js"></script>
+<div id="log"></div>
+<h2>Precondition</h2>
+<ol>
+  <li>
+    Disable the Gyroscope Sensor or run test on a device without Gyroscope Sensor.
```

Comment:
This requires a manual step and access to the device under test (to disconnect the sensor). It will fail otherwise on browsers that correctly implement the spec. Wondering if there isn't a way to indicate that. Or maybe just mark it as a manual test?

Summoning @jgraham, @Ms2ger and @gsnedders for advice here.